### PR TITLE
Removed address book, updated tests, modified UI

### PR DESCRIPTION
**Pull request by CharmaineKoh**

### CODE REVIEW

File: src/test/java/nasa/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    public void execute_storageThrowsIoException_throwsCommandException() {
+        // Setup LogicManager with JsonNasaBookIoExceptionThrowingStub
+        JsonNasaBookStorage nasaBookStorage =
+                new JsonNasaBookIoExceptionThrowingStub(temporaryFolder.resolve("ioExceptionNasaBook.json"));
+        JsonUserPrefsStorage userPrefsStorage =
+                new JsonUserPrefsStorage(temporaryFolder.resolve("ioExceptionUserPrefs.json"));
+        StorageManager storage = new StorageManager(nasaBookStorage, userPrefsStorage);
+        logic = new LogicManager(model, storage);
+
+        // Execute add command
```
Comment (kester-ng) [inline src/test/java/nasa/logic/LogicManagerTest.java]: Hi, why is this commented out?

File: src/test/java/nasa/logic/commands/AddCommandIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package nasa.logic.commands;
+
```
Comment (kester-ng) [inline src/test/java/nasa/logic/commands/AddCommandIntegrationTest.java]: can remove this, the addModulecommand test already does this

File: src/test/java/nasa/logic/commands/CommandTestUtil.java [hunk 3]
```diff
@@ -74,6 +73,20 @@
     public static final String PREAMBLE_WHITESPACE = "\t  \r  \n";
     public static final String PREAMBLE_NON_EMPTY = "NonEmptyPreamble";
```
Comment (kester-ng) [inline src/test/java/nasa/logic/commands/CommandTestUtil.java]: if this is not needed, can just remove

File: src/test/java/nasa/logic/commands/DeleteCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
+package nasa.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
```
Comment (kester-ng) [inline src/test/java/nasa/logic/commands/DeleteCommandTest.java]: Remove this test, we already have two delete tests, one for module and one for activity already.

File: src/test/java/nasa/logic/commands/FindCommandTest.java [hunk 5]
```diff
@@ -1,33 +1,25 @@
...
     public void equals() {
-        ActivityContainsKeyWordsPredicate firstPredicate =
-            new ActivityContainsKeyWordsPredicate(Collections.singletonList("first"));
-
-        ActivityContainsKeyWordsPredicate secondPredicate =
-            new ActivityContainsKeyWordsPredicate(Collections.singletonList("second"));
+        NameContainsKeywordsPredicate firstPredicate =
+                new NameContainsKeywordsPredicate(Collections.singletonList("first"));
+        NameContainsKeywordsPredicate secondPredicate =
+                new NameContainsKeywordsPredicate(Collections.singletonList("second"));
```
Comment (kester-ng) [inline src/test/java/nasa/logic/commands/FindCommandTest.java]: please revert the changes of this, thanks.

File: src/test/java/nasa/logic/parser/FindCommandParserTest.java [hunk 6]
```diff
@@ -20,15 +16,16 @@ public void parse_emptyArg_throwsParseException() {
...
     @Test
     public void parse_validArgs_returnsFindCommand() {
         // no leading and trailing whitespaces
-        FindCommand findCommand =
-            new FindCommand(new ActivityContainsKeyWordsPredicate(Arrays.asList("Tutorial", "Exam")));
-        assertParseSuccess(parser, "Tutorial Exam", findCommand);
-
-        //multiple whitespaces in-between words
-        assertParseSuccess(parser, "\n Tutorial \n \t Exam \t", findCommand);
+        FindCommand expectedFindCommand =
```
Comment (kester-ng) [inline src/test/java/nasa/logic/parser/FindCommandParserTest.java]: Same thing, please revert, if not this test is not useful

File: src/test/java/nasa/model/ModelManagerTest.java [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+    }
+
+    @Test
+    public void hasModule_personInNasaBook_returnsTrue() {
+        modelManager.addModule(GEH1001);
+        assertTrue(modelManager.hasModule(GEH1001));
+    }
+
+    @Test
+    public void getFilteredPersonList_modifyList_throwsUnsupportedOperationException() {
```
Comment (don-tay) [inline src/test/java/nasa/model/ModelManagerTest.java]: Small issue with naming of this method.
